src/core_plugins/console/public/src/directives/help.html

```diff
@@ -1,36 +1,42 @@
-<h3>Help</h3>
+<div class="localDropdownContent localDropdownSection">
```

Comment:
Each app has a bunch of partials for the dropdowns that show up when you click on a menu button. I've made the same kinds of changes to each one of these:

1. Wrapped it in an element with the `localDropdownContent` class and sometimes `localDropdownSection`. `localDropdownContent` adds the light gray styling and some padding around the element. `localDropdownSection` is typically used as a child of this element but can sometimes be applied to it directly; it just adds a margin at the bottom.
2. Add a title element if it's missing, using the `localDropdownTitle` class.
3. If there are tabs, use the `list-group-item list-group-item--noBorder` classes. The first class is from Bootstrap, and the second one is a modifier I made to tweak it slightly. I made this change just to make our tabs look consistent, for now. Later, we'll have our own tabs components to replace this.